application-admintools-api/src/main/java/com/xwiki/admintools/jobs/CustomJobResult.java

```diff
@@ -17,26 +17,26 @@
  * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
  */
-package com.xwiki.admintools.health;
+package com.xwiki.admintools.jobs;
 
 import java.util.List;
 
 import org.xwiki.stability.Unstable;
 
 /**
- * Result of a health check. Stores a custom message for the summary of the result, the severity level represented by
- * {@link HealthCheckResultLevel}. The result may also contain a {@link List} of additional parameters used if there
- * is need to store more information about the result.
+ * Result of a job. Stores a custom message for the summary of the result, the severity level represented by
+ * {@link CustomJobResultLevel}. The result may also contain a {@link List} of additional parameters used if there is
+ * need to store more information about the result.
  *
  * @version $Id$
  * @since 1.0
  */
 @Unstable
-public class HealthCheckResult
+public class CustomJobResult
```

Comment:
`Custom` is general enough that it's at the same time good, because it adds a suffix so you can differentiate the job, but also bad, because it doesn't actually adds a particularity for this Job results class (anything can be custom). I know these classes should be general, in case the API will be used by someone else, but you still need a particularity
What about `AdminJobResult`? So you don't actually link it to `AdminTools`